### ISSUE TITLE
vuestic-select with object

### ISSUE THREAD

**Issue by matheus073** [epicmaxco/vuestic-admin#543]
How do i use vuestic-select with an object, specifying what should appear to the user and the value that he should get.

example:

Object: [
        { name: 'Secure', type: 1 },
        { name: 'Insecure', type: 2 },
],

In user should return **name** and for me should return **type**

**Comment by asvae** [comment]
The easiest would be to wait for 2.0.

You can also use computed and map these options. Should work with current select.

Another solution would be to use [vue-multiselect](https://vue-multiselect.js.org/) - an excellent library. Select in 2.0 will be heavily similar.

**Comment by matheus073** [comment]
**Thanks**

**Comment by yamanakata** [comment]
This is not implemented in version 2.0, right?

**Comment by fabus961** [comment]
It's implemented in the current Version - see [here](https://vuestic.dev/de/ui-elements/select#objects-as-options).

```
<template>
  <div style="max-width: 300px;">
    <va-select class="mb-6" label="Default" v-model="value" :options="options" />

    <va-select class="mb-6" label="Text by" v-model="value" :options="options" text-by="textBy" />

    <va-select class="mb-6" label="Text by (function)" v-model="value" :options="options" :text-by="(option) => option.textBy" />

    <va-select class="mb-6" label="value by" v-model="value" :options="options" value-by="valueBy" />

    <va-select class="mb-6" label="value by (function)" v-model="value" :options="options" :value-by="(option) => option.valueBy" />

    <va-alert color="info">
      <template #title>
        Value
      </template>
      {{ value }}
    </va-alert>
  </div>
</template>
```

```
<script>
export default {
  data () {
    const options = [
      {
        text: 'First',
        textBy: 'First text by',
        value: '1',
        valueBy: 'first',
      },
      {
        text: 'Second',
        textBy: 'Second text by',
        value: '2',
        valueBy: 'second',
      },
      {
        text: 'Third',
        textBy: 'Third text by',
        value: '3',
        valueBy: 'third',
      },
    ]

    return {
      value: options[0],
      options,
    }
  },
}
</script>
```
